app/components/extension-buttons.ts

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+export default function(app) {
+    app.component('extensionButtons', {
+      controller: [
+        '$scope', '$element', '$attrs', '$window',
+        function($scope, $element, $attrs, $window) {
+          this.isChromium = !!$window.chrome;
+          this.isFirefox = $window.navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+
+          // from
+          // https://developer.mozilla.org/en/docs/Installing_Extensions_and_Themes_From_Web_Pages
+          $scope.firefoxAddonInstall = function(aEvent) {
+            for ( let a = aEvent.target; a.href === undefined; ) a = a.parentNode;
+            const params = {
+              'PaperHive': {
+                URL: aEvent.target.xpiUrl,
+                IconURL: aEvent.target.getAttribute('iconURL'),
+                toString: function () { return this.URL; }
+              }
+            };
+            $window.InstallTrigger.install(params);
+            return false;
+          };
+        }],
+        template: `
+          <div ng-if="$ctrl.isChromium">
+            <a class="btn btn-primary btn-block ph-margin-bottom-10"
+               href="https://chrome.google.com/webstore/detail/paperhive/fihafdlllifbanclcjljledeifcdjbok"
+               onclick="chrome.webstore.install();return false;"
+               >
+              <img
+                alt="Chrome logo"
+                width="30px"
+                src="./static/img/chrome.svg"
+                >
+              Add to Chrome
+            </a>
+            <a class="ph-font-size-small"
```

Comment:
Let's use [`<small>` or the `small` class](http://getbootstrap.com/css/#small-text) here.